src/parse.par

```diff
@@ -692,7 +692,7 @@ ast_node<char*>		:	'=' identifier		[* @@ = pstrdup( strbuf ); *]
 
 production<PROD*>	:	line_number rhs_opt:rhs code_opt_dup:act
 							ast_node
-							prod_directives*
+							prod_directives?
```

Comment:
Hello @mingodad,
as already noted [here](/phorward/unicc/tree/master/targets), `prod_directives` should remain to be allowed multiple times. I think the real bugfix of your problem is the one in the next line! Can you please re-check your grammar by using `prod_directives*` still?

Reply:
I don't see any mention to `prod_directives` on the link you've pointed out.
As I understand if we have `prod_directives*` then the user can write something as shown bellow, and it doesn't make sense for me:
```
expr:
  expr '*' expr #precedence tok1 #precedence tok2 #precedence tok3 ...;
```
But if we have `prod_directives+` then we'll get a compile time error if the user try to write more than on precedence directive.

Reply:
Hello @mingodad, the syntax should exactly allow for this:
```
expr:
  expr '*' expr #precedence tok1 #precedence tok2 #precedence tok3 ...;
```
When there might be another production directive implemented in future, your use of `prod_directives?` only allows to specify exactly one directive, and specifying e.g.
```
expr:
  expr '*' expr #precedence tok1 #emit mul;
```
won't be allowed.

Therefore please change this line back to `prod_directives*`.